packages/flutter/lib/src/material/mergeable_material.dart

```diff
@@ -122,10 +122,10 @@ class MergeableMaterial extends StatefulWidget {
   final bool hasDividers;
 
   @override
-  String toString() {
```

Comment:
I assume this toString was added by accident and an int value is displayed with toStringAsFixed by accident as well.
The base class should be adding the key if it exists so there is no need to add it again here.
 Correct me if I'm wrong and I'll look at adding an option to display ints using toStringAsFixed. E.g. so elevation shows as 3.0 instead of 3.